src/tcgwars/logic/impl/gen8/DarknessAblaze.groovy

```diff
@@ -1092,6 +1092,34 @@ public enum DarknessAblaze implements LogicCardInfo {
           attackRequirement {}
           onAttack {
             damage 130
+            afterDamage {
```

Comment:
Is this mess at all accurate? I'm trying to emulate what a retreat cost would do, so if you return an energy that gives 3 RAINBOW, it should satisfy the 2 [W] requirement. If this is incorrect I can use what some other cards in the code base do.